src/main/resources/META-INF/plugin.xml

```diff
@@ -141,11 +143,15 @@
 
         <fileBasedIndex implementation="org.idea.plugin.atg.index.AtgComponentsIndexExtension"/>
         <fileBasedIndex implementation="org.idea.plugin.atg.index.AtgXmlsIndexExtension"/>
+        <fileBasedIndex implementation="org.idea.plugin.atg.index.XmlActorIndexExtension"/>
 
         <facetType implementation="org.idea.plugin.atg.module.AtgModuleFacetType"/>
         <framework.detector implementation="org.idea.plugin.atg.framework.AtgFrameworkDetector"/>
         <iconProvider implementation="org.idea.plugin.atg.module.AtgIconProvider" id="atgFolders" order="first"/>
 
+        <defaultLiveTemplatesProvider implementation="org.idea.plugin.atg.template.live.LiveTemplateProvider"/>
+        <implicitUsageProvider implementation="org.idea.plugin.atg.usage.implicit.ActorImplicitUsageProvider"/>
```

Comment:
Will be grateful if these capabilities will be described, what's expected.
As ex, I didn't face with Atg actors before, so don't aware of what can be improved and in what way
Ideally - as Issue in GitHub. Thanks.